test/test_mypy.py

```diff
@@ -70,6 +74,7 @@ def setUpClass(cls) -> None:
     def tearDownClass(cls) -> None:
         cls.client.close()
 
+    @client_context.require_no_auth
```

Comment:
is `client_context.require_no_auth` needed in this tests?

Reply:
Yes, it was failing without it

Reply:
Trying without it on this run now that we're using `IntegrationTest`

Reply:
Cool, not needed anymore